Plan Chess From Outer Space/Assets/Scripts/ChessPiece.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+// Enum containing all the possible chess pieces
+public enum ChessPieceType
+{
+    Pawn,
+    Knight,
+    Bishop
+}
+
+public class ChessPiece : MonoBehaviour
+{
+    // --------------- Member variables and data --------------- //
+    public ChessPieceType PieceType;
```

Comment:
Maybe we should ditch this for virtual/abstract properties and methods? I think we can call them in other scripts with a reference obtained through a `GameObject.GetComponent<ChessPiece>` call. See also [this forum post](https://forum.unity.com/threads/calling-function-from-other-scripts-c.57072/).

Reply:
> Should we drop this and turn to virtual/abstract properties and methods?

I think that's a good idea. But again, this PR is for the chess board feature, we should design a better ChessPiece class in another PR.